src/global/core-styles.scss

```diff
@@ -1,36 +1,5 @@
 // These are exported for external use.
 
+@import '../style/theme-color-variables.scss';
 @import '../style/colors.scss';
 @import '../style/shadows.scss';
-
-:root {
```

Comment:
I don't think your fixup will work, because it needs to register the variables on `:root` in this file, but on `:host` in the other file, and with your fixup, it uses `:host` in both cases.

That's why I hadn't made a single source of truth, because I don't really know how to do that in this case, and didn't want to spend too much time trying. But I think a scss mixin could probably do it?